spring-petclinic-customers-service/pom.xml

```diff
@@ -110,4 +125,28 @@
             </build>
         </profile>
     </profiles>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.hibernate.orm.tooling</groupId>
+                <artifactId>hibernate-enhance-maven-plugin</artifactId>
+                <version>5.2.17.Final</version>
```

Comment:
i'm open to suggestions on how to manage this build dependency - as it's required for hibernate to run in jdk9+ at runtime (tests will pass without it). i've tried a few things here:

* placing this stanza in parent pom didn't work
* referencing version info from parent pom didn't work
* leaving the version out generates a warning
* placing this inside the docker build plugin section means non-docker builds will generate jars which won't work during run time